File: src/Marvin.Runtime.Maintenance.Web.UI/src/databases/api/requests/ExecuteScriptRequest.ts
/*
 * Copyright (c) 2020, Phoenix Contact GmbH & Co. KG
 * Licensed under the Apache License, Version 2.0
*/

import DatabaseConfigModel from "../../models/DatabaseConfigModel";
import ScriptModel from "../../models/ScriptModel";

export default class ExecuteScriptRequest
{
    Config : DatabaseConfigModel;
    Script : ScriptModel;
}

File: src/Moryx.Runtime.Maintenance/Plugins/Databases/Wcf/Request/ExecuteScriptRequest.cs
// Copyright (c) 2020, Phoenix Contact GmbH & Co. KG
// Licensed under the Apache License, Version 2.0

namespace Moryx.Runtime.Maintenance.Plugins.Databases
{
    /// <summary>
    /// Request to execute a database script
    /// </summary>
    public class ExecuteScriptRequest
    {
        /// <summary>
        /// Configuration of the database
        /// </summary>
        public DatabaseConfigModel Config { get; set; }

        /// <summary>
        /// Script to be executed
        /// </summary>
        public ScriptModel Script { get; set; }
    }
}
